re.c

```diff
@@ -503,14 +503,16 @@ rb_reg_source(VALUE re)
 }
 
 /*
- * call-seq:
- *    rxp.inspect   -> string
+ *  call-seq:
+ *    inspect -> string
+ *
+ *  Returns a nicely-formatted string representation of +self+:
+ *
+ *    /ab+c/ix.inspect # => "/ab+c/ix"
  *
- * Produce a nicely formatted string-version of _rxp_. Perhaps surprisingly,
- * <code>#inspect</code> actually produces the more natural version of
- * the string than <code>#to_s</code>.
+ *  Contrast with Regexp#to_s:
```

Comment:
I realize it was already there, but I would drop this part and the example below, as `Regexp#to_s` behavior is covered in it's own documentation.

Reply:
@jeremyevans, change to Related:?

Reply:
Sure, `Related: #to_s` seems fine.